src/main/resources/hudson/plugins/s3/S3BucketPublisher/config.jelly

```diff
@@ -34,6 +34,10 @@
     <f:checkbox title="Don't wait for completion of concurrent builds before publishing to S3" />
   </f:entry>
 
+  <f:entry field="dontSetBuildResultOnFailure" title="">
+    <f:checkbox title="Do Not Set Build Result on Failure" />
```

Comment:
Please, add help as well (e.g. look at `/plugin/copyartifact/help-buildSelector.html`).
Otherwise it's not obvious that after setting this checkbox user will receive exception.

Reply:
Documentation added. please review.